java/code/src/com/suse/manager/utils/SaltUtils.java

```diff
@@ -1040,20 +1091,22 @@ private void handleSupportData(JsonElement jsonResult, ServerAction serverAction
         var action = (SupportDataAction)serverAction.getParentAction();
         var caseNumber = action.getDetails().getCaseNumber();
         serverAction.getServer().asMinionServer().ifPresentOrElse(minionServer -> {
-            var bundleName = caseNumber + "_" + action.getId() + "_" + minionServer.getMinionId() + ".tar";
-            //TODO: move action path function to more general place
-            var actionPath = Paths.get(Config.get().getString(ConfigDefaults.MOUNT_POINT))
-                    .resolve(ScapFileManager.getActionPath(action.getOrg().getId(),
-                            minionServer.getId(), action.getId()));
-            var bundle = actionPath.resolve(bundleName);
+            var actionPath = MinionActionUtils.getFullActionPath(action.getOrg().getId(), minionServer.getId(),
+                    action.getId());
+            var bundle = actionPath.resolve("bundle.tar");
+            var dateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd_hhmm").withZone(ZoneOffset.UTC);
+            var proxy = minionServer.isProxy();
+            var prefix = proxy ? "_P_" : "_M_";
+            var uploadName = "SR%s%s_%s_%s.tar"
+                    .formatted(prefix, caseNumber, minionServer.getMinionId(), dateTimeFormatter.format(Instant.now()));
```

Comment:
- should we really use the minion id or better getHostname() ? The minion id can be something completely different.
- what about a 3 char prefix (_PXY_ and _MGR_). I think this is more visible than just P or M.
- should the case number appear after SR ?